tools/src/test/java/org/apache/kafka/tools/reassign/ReassignPartitionsCommandTest.java

```diff
@@ -128,18 +126,6 @@ public void testReassignment() throws Exception {
         executeAndVerifyReassignment();
     }
 
-    @ClusterTests({
-        @ClusterTest(types = {Type.KRAFT, Type.CO_KRAFT}, metadataVersion = IBP_3_3_IV0)
-    })
-    public void testReassignmentWithAlterPartitionDisabled() throws Exception {
-        // Test reassignment when the IBP is on an older version which does not use
-        // the `AlterPartition` API. In this case, the controller will register individual
-        // watches for each reassigning partition so that the reassignment can be
-        // completed as soon as the ISR is expanded.
```

Comment:
Has the code for this fallback been removed? If not, do we have a JIRA ticket for it? If not, I'll file one.

Reply:
@cmccabe Do you know?

Reply:
It seems to me the behavior is related to zk only, and in kraft mode `AlterPartition` should be supported totally. Hence, it is fine to remove the test.

BTW, the test case for kraft is unnecessary - I overlook the scenario in merging #15675

Reply:
Makes sense. To confirm, you're saying we can remove this test _and_ we don't need any other follow-up since the comment was for the zk controller. Right?

Reply:
>  you're saying we can remove this test and we don't need any other follow-up since the comment was for the zk controller. Right?

we are on the same page!